projects/plugins/jetpack/class.jetpack.php

```diff
@@ -4952,6 +4998,25 @@ public static function authorize_ending_authorized( $data ) {
 		self::handle_post_authorization_actions( $activate_sso, $do_redirect_on_error );
 	}
 
+	/**
+	 * Fires on the jetpack_site_registered hook and acitvates default modules
+	 */
+	public static function site_registered() {
+		$active_modules = Jetpack_Options::get_option( 'active_modules' );
+		if ( $active_modules ) {
+			self::delete_active_modules();
+
+			// If there was previously activated modules (a reconnection), re-activate them all including those that require a user, and do not re-activate those that have been deactivated.
+			self::activate_default_modules( 999, 1, $active_modules, null, null );
```

Comment:
```suggestion
			self::activate_default_modules( 999, 1, $active_modules );
```
These are probably excessive ¯\\\_(ツ)\_/¯